documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -94,6 +95,19 @@ spec:
           patternType: literal
         operation: Describe
         host: "*"
+      - resource:
+          type: topic
+          name: my-topic
+          patternType: literal
+        operation: AlterConfigs
+        host: "*"
+      # access to the cluster
+      - resource:
+          type: cluster
+          name: my-cluster
```

Comment:
I don't think this needs the name and patternType, or?

Reply:
I tested it and looks like topic name was necessary

Reply:
But it is not a topic name. If it is required, what is the name which should be there? The Kafka cluster has no real name.

Reply:
Oh, you were actually comneting on something else and not on this one. I think the ACLs look good now.